stdlib/asm/crypto/fri/frie2f4.masm

```diff
@@ -90,7 +100,7 @@ end
 #! Input:  [layer_ptr, layer_ptr, poe, p, e1, e0, layer_ptr, rem_ptr, x, x, x, x, x, x, x, x, ...]
 #! Output: [layer_ptr+8, layer_ptr+8, poe^4, f_pos, ne1, ne0, layer_ptr+8, rem_ptr, x, x, x, x, x, x, x, x, ...]
```

Comment:
```suggestion
#! Output: [has_next_iteration, layer_ptr+8, layer_ptr+8, poe^4, f_pos, ne1, ne0, layer_ptr+8, rem_ptr, x, x, x, x, x, x, x, x, ...]
```

or something similar

Reply:
Good catch, used `is_not_last_layer` as it is closer to the condition we are using in the loop